# Card types

## Rule cards

Rule cards explain how to use the **Enterprise Architecture Pitch Deck** for [Work](../rules/work.md) or [Play](../rules/play.md).

```

Card image

```

## :fontawesome-solid-users:{: .role } Role cards 

Each player is given a role card. This represents your character in the game. For most of the game you are an Enterprise Architect. You have a special power based on the knowledge you've acquired over your career. 

During the [Pitch](../rules/pitch.md) stage of the game you'll temporarily become another character - a member of the Executive.


```

Card image front and back

```


## :fontawesome-solid-exclamation-triangle:{: .challenge } Challenge cards 

Challenge cards represent the challenge your trying to solve for your organisation. The Pitch Deck you create will need to explain how to recommend solving the challenge.  

```

Card image front and back

```


## :fontawesome-solid-chart-bar:{: .leanix } Report cards 

Report cards are used to build your Pitch Deck.

```

Card image front and back

```


##  :fontawesome-solid-calendar-times:{: .event  } Event cards

Event cards add an element of chance to the game. Good and bad things can happen.

```

Card image front and back

```


## :fontawesome-solid-star:{: .kudos }Kudo cards 

Kudos cards add or subtract from your score. 

```

Card image front and back - Trusted Advisor on one side, Over-promise Under-deliver 

```


## Pitch cards 

:fontawesome-solid-map-marker-alt:{: .situation }  :fontawesome-solid-exclamation-triangle:{: .challenge }  :fontawesome-solid-lightbulb:{: .recommendation }  :fontawesome-solid-bullhorn:{: .call-to-action }

Add pitch cards to start and end of your Pitch Deck when you're:

- creating a deck for [work](../rules/work.md)
- getting ready to [pitch](../rules/pitch.md) to the Executive during [play](../rules/play.md)


```

Images of Situation, Challenge, Recommendation, Call-to-Action cards

```

## The About card

Use the [About card](../about/index.md) to: 

- Contact us
- Earn real kudos for your innovative custom reports
- Reuse or adapt these cards using a Creative Commons licence
- Read the origin story

~004~
