# Card types

## Rule cards

Rule cards explain how to use the **Enterprise Architecture Pitch Deck** for [Work](work.md) or [Play](play.md).

```

Card image

```

## Role cards :fontawesome-solid-user: 

Each player is given a role card. This represents your character in the game. For most of the game you are an Enterprise Architect. You have a special power based on the knowledge you've acquired over your career. 

During the [Pitch](pitch.md) stage of the game you'll temporarily become another character - a member of the Executive.


```

Card image front and back

```

## Pitch cards 

Add pitch cards to your Pitch Deck to start and end your presentation.

:fontawesome-solid-map-marker-alt:{: .situation }  :fontawesome-solid-lightbulb:{: .recommendation } :fontawesome-solid-bullhorn:{: .call-to-action }

```

Images of Situation, Challenge, Recommendation, Call-to-Action cards

```

## Challenge cards :fontawesome-solid-exclamation-triangle:{: .challenge }

Challenge cards represent the challenge your trying to solve for your organisation. The Pitch Deck you create will need to explain how to recommend solving the challenge.  

```

Card image front and back

```


## Report cards :fontawesome-solid-chart-bar:{: .leanix } 

Report cards are used to build your Pitch Deck.

```

Card image front and back

```


## Event cards :fontawesome-solid-calendar-times:{: .event  }

Event cards add an element of chance to the game. Good and bad things can happen.

```

Card image front and back

```


## Kudo cards :fontawesome-solid-star:{: .kudos }

Kudos cards add or subtract from your score. 

```

Card image front and back - Trusted Advisor on one side, Over-promise Under-deliver 

```

## About us card

Learn our [origin story](about/), contact us or reuse and adapt these cards using a Creative Commons licence.
